ETC.tsv

```diff
@@ -7856,7 +7856,7 @@ ETC_20150317_007856	{memo X}The third Thorn Gate (2)
 ETC_20150317_007857	{memo X}The third Thorn Gate (3)
 ETC_20150317_007858	{memo X}The first Thorn Gate (3)
 ETC_20150317_007859	{memo X}Melt a thorn tunnel to use an acidity solution
-ETC_20150317_007860	Use the Magic Crystal to remove the thorny vines
+ETC_20150317_007860	Use the Magic Crystal to remove the thorn vines
```

Comment:
I think thorny  works better for this, also for all other thorn vines (unless its an item)